Packages/CommonUI/CommonUI/Classes/View/Picker/ImagePicker/ImagePicker.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ImagePicker.swift
+//  CommonUI
+//
+//  Created by MinhDev on 05/07/2022.
+//
+
+import SwiftUI
+import Photos
+
+private enum Constants {
+	static let padding = 15.0
+	static let sizeIcon = 30.0
+	static let paddingTop = 50.0
+	static let paddingHorizontal = 17.0
+}
+
+public struct ImagePicker: View {
+	
+	@Environment(\.presentationMode) var presentationMode
+	@Environment(\.colorScheme) var colorScheme
+	
+	var doneAction: ([SelectedImageModel]) -> Void
+	
+	@StateObject private var viewModel = ImagePickerViewModel()
+	
+	private var columns = Array(repeating: GridItem(.flexible(), spacing: 0), count: 2)
+	// MARK: - Init
+	public init(doneAction: @escaping ([SelectedImageModel]) -> Void) {
+		self.doneAction = doneAction
+	}
+	
+	public var body: some View {
+		VStack {
+			headerView
+			GeometryReader { geometryReader in
+				ScrollView(.vertical, showsIndicators: false) {
+					LazyVGrid(columns: columns, spacing: 0) {
+						ForEach($viewModel.fetchedImages) { $photo in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/CommonUI/CommonUI/Classes/View/Picker/ImagePicker/ImagePicker.swift#L39"></span>Unused parameter "$photo" in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>ImagePicker.swift:39</code></td>
    </tr>
  </tbody>
</table>